setup.py

```diff
@@ -9,5 +9,22 @@
   url = 'https://github.com/coala-analyzer/coala-decorators',
   download_url = 'https://www.github.com/coala-analyzer/coala-decorators/tarball/0.1.1',
   keywords = ['coala', 'decorators', 'bears'],
-  classifiers = [],
+  classifiers = ['Development Status :: 4 - Beta',
+
+                 'Environment :: Console',
+                 'Environment :: MacOS X',
+                 'Environment :: Win32 (MS Windows)',
+
+                 'Intended Audience :: Science/Research',
+                 'Intended Audience :: Developers',
+
+                 'License :: OSI Approved :: MIT License',
+
+                 'Operating System :: OS Independent',
+
+                 'Programming Language :: Python :: Implementation :: CPython',
+                 'Programming Language :: Python :: 3.3',
+                 'Programming Language :: Python :: 3.4',
+                 'Programming Language :: Python :: 3.5',
+                 'Programming Language :: Python :: 3 :: Only'],
 )
```

Comment:
Wrong indentation ?
What sort of code convention do you wish to adopt in this repo ? (If it's same as coala, then the round bracket should be on the line above)